BrainPortal/app/views/portal/_available_datasets_table.html.erb

```diff
@@ -41,7 +41,10 @@
     end
   %>
   <tr>
-    <td><%= overlay_description((name + "\n" + description).strip) %></td>
+    <td>
+      <%= name %>
+      <%= closable_overlay_content_link("(i)", name, description) %>
```

Comment:
Instead of the string `"(i)"` use `" &#8505; ".html_safe` and provide the class option "action_link" to add that CSS class to the link (assuming the helper has been modified according to my previous comment). It will make the links much nicer.